src/scrape_up/gitlab/__init__.py

```diff
@@ -0,0 +1,4 @@
+from scrape_up.gitlab.users import Users
+from scrape_up.gitlab.project import Project
```

Comment:
Mention all the Class of the `scrape_up.gitlab`. 
![image](https://github.com/Clueless-Community/scrape-up/assets/93156825/f751813d-adfd-49f8-9b42-55d4d233ce98)